src/Form/Theme.php

```diff
@@ -9,8 +9,8 @@
 
 namespace Give\Form;
 
-use function Give\Form\Theme\getActiveID;
-use function Give\Form\Theme\get;
+use InvalidArgumentException;
+use function Give\Helpers\Form\Theme\get;
```

Comment:
When importing a function with a name that generic I suggest adding an alias for clarity:
```php
use function Give\Helpers\Form\Theme\get as getTheme;
```

Reply:
fixed: https://github.com/impress-org/givewp/compare/045cd0436d8c...7c2a09061baa